package.json

```diff
@@ -41,6 +41,7 @@
         "@storybook/react-vite": "^8.6.11",
         "@swc-node/register": "^1.10.10",
         "@swc/core": "1.11.13",
+        "@swc/helpers": "^0.5.17",
```

Comment:
I saw warnings that a few files couldn't resolve this module. I'm not sure why it didn't happen earlier, but adding this to ensure our builds are correct.